include/volume/volume_cooling_gaussians_mass_matrix.hpp

```diff
@@ -0,0 +1,528 @@
+// VolEsti (volume computation and sampling library)
+
+// Copyright (c) 2012-2020 Vissarion Fisikopoulos
+// Copyright (c) 2018-2020 Apostolos Chalkis
+
+//Contributed and/or modified by Apostolos Chalkis, as part of Google Summer of Code 2018 program.
+
+// Licensed under GNU LGPL.3, see LICENCE file
+
+#ifndef VOLUME_COOLING_GAUSSIANS_HPP
+#define VOLUME_COOLING_GAUSSIANS_HPP
+
+//#define VOLESTI_DEBUG
+
+#include <iterator>
+#include <vector>
+#include <list>
+#include <math.h>
+#include <chrono>
+
+#include "cartesian_geom/cartesian_kernel.h"
+#include "random_walks/gaussian_helpers.hpp"
+#include "random_walks/gaussian_ball_walk.hpp"
+#include "random_walks/gaussian_cdhr_walk.hpp"
+#include "random_walks/gaussian_hamiltonian_monte_carlo_exact_walk.hpp"
+#include "sampling/random_point_generators.hpp"
+#include "volume/math_helpers.hpp"
+#include "Eigen/Eigen"
+
+
+/////////////////// Helpers for random walks
+
+template <typename WalkType>
+struct update_delta
+{
+    template <typename NT>
+    static void apply(WalkType, NT) {}
+};
+
+template <typename Polytope, typename RandomNumberGenerator>
+struct update_delta<GaussianHamiltonianMonteCarloExactWalk::Walk<Polytope, RandomNumberGenerator>>
```

Comment:
is this redefined here https://github.com/GeomScale/volesti/pull/230/files#diff-f34df8328972aebd7b85669d7cf3568caf3720a784967874cde9eb497c9afc23R40  or I am missing something?